doc/concepts/configuration/configuration_persistence.rst

```diff
@@ -0,0 +1,233 @@
+..  _configuration_persistence:
+
+Persistence
+===========
+
+To ensure data persistence, Tarantool provides the abilities to:
+
+*   Record each data change request into a :ref:`write-ahead log <internals-wal>` (WAL) file (``.xlog`` files).
+*   Take :ref:`snapshots <internals-snapshot>` that contain on-disk copy of the entire data set for a given moment
+    (``.snap`` files). It is possible to set automatic snapshot creation using the :ref:`checkpoint daemon <configuration_persistence_checkpoint_daemon>`.
+
+During the recovery process, Tarantool can load the latest snapshot file and then read the requests from the WAL files,
+produced after this snapshot was made.
+
+To learn more about the persistence mechanism in Tarantool, see the :ref:`Persistence <concepts-data_model-persistence>` section.
+The formats of WAL and snapshot files are described in detail in the :ref:`File formats <internals-data_persistence>` section.
+
+..  _configuration_persistence_snapshot:
+
+Configure the snapshots
+-----------------------
+
+**Example on GitHub**: `snapshot <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/config/instances.enabled/persistence_snapshot>`_
+
+This section describes how to define snapshot settings in the :ref:`snapshot <configuration_reference_snapshot>` section of a YAML configuration.
+
+..  _configuration_persistence_snapshot_creation:
+
+Configure the automatic snapshot creation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In Tarantool, it is possible to set automatic :ref:`snapshot creation </reference/reference_lua/box_snapshot>`.
+To enable it, the :ref:`snapshot.by.interval <configuration_reference_snapshot_by_interval>` option is used.
```

Comment:
It is enabled by default. The 'to enable it, <...>' sentence may be confusing in this regard.

Reply:
I agree with you. Thanks, fixed!